agenta-backend/agenta_backend/routers/testset_router.py

```diff
@@ -1,27 +1,42 @@
-from fastapi import HTTPException, APIRouter, UploadFile, File, Form, Body
-from agenta_backend.services.db_mongo import testsets
+import csv
+import json
+from copy import deepcopy
+from bson import ObjectId
+from datetime import datetime
+from typing import Optional, List
+
+from fastapi import HTTPException, APIRouter, UploadFile, File, Form, Depends
+
 from agenta_backend.models.api.testset_model import (
     UploadResponse,
     DeleteTestsets,
     NewTestset,
 )
-from datetime import datetime
-from typing import Optional, List
-from bson import ObjectId
-import csv
-import json
+from agenta_backend.config import settings
+from agenta_backend.models.db_models import TestSetDB
+from agenta_backend.services.db_manager import engine, query, get_user_object
+
 
 upload_folder = "./path/to/upload/folder"
 
 router = APIRouter()
 
 
+if settings.feature_flag in ["cloud", "ee"]:
```

Comment:
can you add "demo" here. Otherwise it won't work in the demo

Reply:
Sure, will do!